docs/source/developer_guide/versioning_policy.md

```diff
@@ -0,0 +1,46 @@
+# Versioning policy
+
+Starting with vLLM 0.7.x, the vLLM Ascend Plugin ([vllm-project/vllm-ascend](https://github.com/vllm-project/vllm-ascend)) project follows the [PEP 440](https://peps.python.org/pep-0440/) to publish matching with vLLM ([vllm-project/vllm](https://github.com/vllm-project/vllm)).
+
+## vLLM Ascend Plugin versions
+
+Each vllm-ascend release will be versioned: `v[major].[minor].[micro][rcN][.postN]` (such as
+`v0.7.1rc1`, `v0.7.1`, `v0.7.1.post1`)
+
+- **Final releases**: will typically be released every **3 months**, will take the vLLM upstream release plan and Ascend software product release plan into comprehensive consideration.
+- **Pre releases**: will typically be released **on demand**, ending with rcN, represents the Nth release candidate version, to support early testing by our users prior to a final release.
+- **Post releases**: will typically be released **on demand** to support to address minor errors in a final release. It's different from [PEP-440 post release note](https://peps.python.org/pep-0440/#post-releases) suggestion, it will contain actual bug fixes considering that the final release version should be matched strictly with the vLLM final release version (`v[major].[minor].[micro]`). The post version has to be published as a patch version of the final release.
+- **Developmental releases**: will be tagged on demand before a final/pre/post releases to help create early releases directly from source code to help with iterative development and verification, typically will not be published to PyPI.
```

Comment:
Not sure if dev tag is needed. More tag will lead more user confusion

Reply:
Agree, this was design to some thing rc for all version, but we can use branch sha to instead.

I will remove it.